### PR TITLE
pipeline: outputs: prometheus-remote-write: update docs with adding options related to aws sigv4 authentication

### PR DESCRIPTION
**Pull request by Claych**

### CODE REVIEW

File: pipeline/outputs/prometheus-remote-write.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ Important Note: The prometheus exporter only works with metric plugins, such as
...
 | http_user            | Basic Auth Username                                                                                                                                                                                                                                                    |           |
 | http_passwd          | Basic Auth Password. Requires HTTP_user to be set                                                                                                                                                                                                                      |           |
+| AWS\_Auth                  | Enable AWS SigV4 authentication                                                                                                                                                                                                                                                                                                    | false     |
```
Comment (PettitWesley) [inline pipeline/outputs/prometheus-remote-write.md:16]: the `FLB_AWS_CREDENTIAL_BASE_CONFIG_MAP` adds more options than just these, please document them

File: pipeline/outputs/prometheus-remote-write.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ Important Note: The prometheus exporter only works with metric plugins, such as
...
+| AWS\_Service               | Specify the AWS service code, i.e. es, xray, etc., of your service, used by SigV4 authentication. Usually can be found in the service endpoint's subdomains, `protocol://service-code.region-code.amazonaws.com`                                                                                                                   | aps          |
+| AWS\_Region                | Specify the AWS region of your service, used by SigV4 authentication                                                                                                                                                                                                                                                               |           |
```
Comment (PettitWesley) [inline pipeline/outputs/prometheus-remote-write.md:18]: same as above, we can be specific => region of your Amazon Managed Service for Prometheus workspace.

File: pipeline/outputs/prometheus-remote-write.md (hunk 1 above)
Comment (Claych) [reply pipeline/outputs/prometheus-remote-write.md:16]: added

File: pipeline/outputs/prometheus-remote-write.md [hunk 3]
```diff
@@ -13,6 +13,9 @@ Important Note: The prometheus exporter only works with metric plugins, such as
...
 | http_passwd          | Basic Auth Password. Requires HTTP_user to be set                                                                                                                                                                                                                      |           |
+| AWS\_Auth                  | Enable AWS SigV4 authentication                                                                                                                                                                                                                                                                                                    | false     |
+| AWS\_Service               | Specify the AWS service code, i.e. es, xray, etc., of your service, used by SigV4 authentication. Usually can be found in the service endpoint's subdomains, `protocol://service-code.region-code.amazonaws.com`                                                                                                                   | aps          |
```
[the remark this answers is not in the document]
Comment (Claych) [reply pipeline/outputs/prometheus-remote-write.md]: yes, updated follows your comments

**Review by matthewfala — APPROVED** [review]
Looks good!

**Review by PettitWesley — APPROVED** [review]

**Review by PettitWesley — APPROVED** [review]
Can be merged after feature release

**Comment by patrick-stephens** [comment]
What feature is this waiting on?

**Comment by PettitWesley** [comment]
@patrick-stephens PR merged here in april: https://github.com/fluent/fluent-bit/pull/6986

It should have gone out in 2.1.0 but there wasn't a full list for that one: https://fluentbit.io/announcements/v2.1.0/

**Comment by PettitWesley** [comment]
I see it; https://github.com/fluent/fluent-bit/tree/v2.1.0/plugins/out_prometheus_remote_write

**Pull request merged**